src/main/groovy/uk/ac/ebi/eva/groovy/commons/EVADatabaseEnvironment.groovy

```diff
@@ -88,6 +90,8 @@ class EVADatabaseEnvironment {
 
         def mc = context.getBean(MongoClient.class)
         def mt = context.getBean(MongoTemplate.class)
+        mt.setReadPreference(ReadPreference.primary())
+        mt.setWriteConcern(WriteConcern.MAJORITY)
```

Comment:
Based on the Spring context and application, these may not be fully enforced. So we are extra cautious here and explicitly setting them.